sdk/tests/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -168,9 +174,14 @@
     }
 
     goodByteFormats.forEach(function(internalformat) {
-        var srcTexFormatsTypes = { internalformat: gl.RGBA, format: gl.RGBA, type: gl.UNSIGNED_BYTE };
+        var srcTexFormatsTypes = { internalformat: gl.RGBA8, format: gl.RGBA, type: gl.UNSIGNED_BYTE };
```

Comment:
Is it worth continuing to test copying from a texture with unsized internal format to a texture with a sized internal format? It's a new situation in the WebGL 2.0 tests.